www/components/Nav/Annoucement.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { IconArrowRight } from '@supabase/ui'
+
+import data from '~/data/Annoucement.json';
+
+type A = {
+  show: boolean;
+  text: string;
+  link: {
+    text: string;
+    url: string;
+  }
+}
+
+const a: A = data;
+
+const Annoucement = () => {
+  if (!a.show) return (<></>)
+  return (
+    <div className="bg-brand-700 dark:bg-gray-700">
+      <div className="flex flex-col items-center justify-center py-2 mx-auto space-x-2 text-white sm:flex-row lg:container lg:px-16 xl:px-20">
```

Comment:
on mobile it looks a little off, if there's any improvements you can make that would be great

![Screenshot 2021-05-18 at 11 50 05](https://user-images.githubusercontent.com/8291514/118587462-3c236d00-b7cf-11eb-960c-8913ec993c97.png)

Reply:
I improved it but both texts mustn't be too long @MildTomato

Reply:
👍